tests/unit_tests/pyanaconda_tests/test_ks_version.py

```diff
@@ -59,7 +59,7 @@ def test_data(self):
 
     def test_dracut_commands(self):
         """Test that dracut uses the right versions of kickstart commands"""
-        # top_srcdir should have been set by nosetests.sh. If it wasn't, the KeyError
+        # top_srcdir should have been set by unit_tests.sh. If it wasn't, the KeyError
```

Comment:
This is changed too early - the file is not renamed yet in that commit - but never mind.

Reply:
Fixed.